bin/resources/GameIndex.yaml

```diff
@@ -39938,9 +39971,6 @@ SLUS-20153:
 SLUS-20154:
   name: "NFL Quarterback Club 2002"
   region: "NTSC-U"
-SLUS-20157:
```

Comment:
Are you sure about this?

Reply:
Yes, it's a typo of SLUS-20517, and SLUS-20157 is not used for any other PS2 game afaik.